src/main/java/co/rsk/bitcoinj/script/ErpFederationRedeemScriptParser.java

```diff
@@ -4,15 +4,14 @@
 
 import co.rsk.bitcoinj.core.Utils;
 import co.rsk.bitcoinj.core.VerificationException;
+import java.math.BigInteger;
 import java.util.ArrayList;
 import java.util.List;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class ErpFederationRedeemScriptParser extends StandardRedeemScriptParser {
     private static final Logger logger = LoggerFactory.getLogger(ErpFederationRedeemScriptParser.class);
-    public static long MAX_CSV_VALUE = 65_535L; // 65535 is 2 bytes. The max value currently accepted
-    public static int CSV_SERIALIZED_LENGTH = 2;
```

Comment:
I don't think MAX_CSV_VALUE should be removed. It was a good barrier against mistakes. What's the MAX/MIN time we want't to lock for the ERP? The consensus will trim it to consider the last 16 bits anyway... so 65_535 seems like an OK max.

Reply:
Actually, it is VERY IMPORTANT the the MAX_CSV_VALUE does not exceed 1 << 22 because that would set that bit to 1 and would make ERP fail for ever (well... tx's would have to be sent with a different sequenceNumber... and then everything would be seconds not blocks... so... not good)